ibis/backends/tests/test_array.py

```diff
@@ -1384,7 +1384,75 @@ def test_zip_unnest_lift(con):
     t = ibis.memtable(data)
     zipped = t.mutate(zipped=t.array1.zip(t.array2))
     unnested = zipped.mutate(unnest=zipped.zipped.unnest())
-    lifted = unnested.unnest.lift()
+    lifted = unnested["unnest"].lift()
     result = con.execute(lifted)
     expected = pd.DataFrame({"f1": [1, 2, 3], "f2": [4, 5, 6]})
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.notimpl(
+    ["datafusion", "pandas", "polars", "dask", "flink"],
+    raises=com.OperationNotDefinedError,
+)
+@pytest.mark.parametrize(
+    "colspec",
+    ["y", lambda t: t.y, ibis._.y],
+    ids=["string", "lambda", "deferred"],
+)
+def test_table_unnest(backend, colspec):
```

Comment:
I'm curious if `table.unnest` works with doubly nested (`array<struct<array<>>`) fields? If so, could you add a test for that case?

Reply:
Added a test case in [`971b005` (#9423)](https://github.com/ibis-project/ibis/pull/9423/commits/971b0052c745a1610d5e8bb29240298024031675). Let me know whether it's sufficient.